packages/frontend/src/pages/about-misskey.vue

```diff
@@ -71,6 +76,24 @@
 					</div>
 					<template #caption><MkLink url="https://github.com/misskey-dev/misskey/graphs/contributors">{{ i18n.ts._aboutMisskey.allContributors }}</MkLink></template>
 				</FormSection>
+				<FormSection>
+					<template #label>{{ i18n.ts._aboutMisskey.forkContributors }}</template>
+					<div :class="$style.contributors">
+						<a href="https://github.com/anatawa12" target="_blank" :class="$style.contributor">
+							<img src="https://avatars.githubusercontent.com/u/22656849?v=4" :class="$style.contributorAvatar">
+							<span :class="$style.contributorUsername">@anatawa12</span>
+						</a>
+						<a href="https://github.com/niwaniwa" target="_blank" :class="$style.contributor">
+							<img src="https://avatars.githubusercontent.com/u/10182706?v=4" :class="$style.contributorAvatar">
+							<span :class="$style.contributorUsername">@niwaniwa</span>
+						</a>
+						<a href="https://github.com/Sayamame-beans" target="_blank" :class="$style.contributor">
+							<img src="https://avatars.githubusercontent.com/u/61457993?v=4" :class="$style.contributorAvatar">
+							<span :class="$style.contributorUsername">@Sayamame-beans</span>
+						</a>
+					</div>
+					<template #caption><MkLink url="https://github.com/misskey-dev/misskey/graphs/contributors">{{ i18n.ts._aboutMisskey.allForkContributors }}</MkLink></template>
```

Comment:
あ、リンク変わってないです

Reply:
```suggestion
					<template #caption><MkLink url="https://github.com/niri-la/misskey.niri.la/graphs/contributors">{{ i18n.ts._aboutMisskey.allForkContributors }}</MkLink></template>
```
ただし、これは本家のcontributorも含むようですね。

Reply:
本家を含むのはgithubの仕様なのでそうって感じですね